test/repo/Makefile

```
@@ -0,0 +1,83 @@
+# This test clones a proof repository and compares the proof results
+# with the proof results after running cbmc-starter-kit-update.
+
+PACKAGE_ROOT = ../..
+
+REPO = https://github.com/FreeRTOS/coreHTTP.git
+REPO_CBMC_ROOT = test/cbmc
+REPO_PROOF_ROOT = $(REPO_CBMC_ROOT)/proofs
+REPO_PROOFS = --proofs \
+  HTTPClient_AddRangeHeader \
+  HTTPClient_ReadHeader \
+  HTTPClient_strerror \
+  findHeaderFieldParserCallback \
+  findHeaderOnHeaderCompleteCallback \
+  findHeaderValueParserCallback \
+  httpParserOnBodyCallback \
+  httpParserOnHeaderFieldCallback \
+  httpParserOnHeaderValueCallback \
+  httpParserOnHeadersCompleteCallback \
+  httpParserOnMessageBeginCallback \
+  httpParserOnMessageCompleteCallback \
+  httpParserOnStatusCallback
+
+REPO_ROOT = /tmp/repo
+LITANI_ROOT = /tmp/litani
+CONFIG_ROOT = /tmp/config
+REPORT_ROOT = /tmp/reports
+REPORT_ROOT1 = $(REPORT_ROOT)1
+REPORT_ROOT2 = $(REPORT_ROOT)2
+
+LITANI = $(LITANI_ROOT)/litani
+CONFIG = $(CONFIG_ROOT)/tests/config/config.py
+
+default:
+	@ echo "Run test with 'make test'"
+
+test: clone-repo clone-litani clone-config run1 install-starter update-starter run2 compare
+
+clone-repo:
+	$(RM) -r $(REPO_ROOT)
```

We might want to add `-f` as well to suppress unnecessary prompts on various Linux distros.
```suggestion
	$(RM) -rf $(REPO_ROOT)
```
(same comment on other `rm`s below.

---

Left as is.  RM is a gnu makefile variable defined to be 'rm -f'.